src/contexts/shoppingListsContext.tsx

```diff
@@ -48,16 +58,60 @@ export const ShoppingListsProvider = ({ children }: ProviderProps) => {
 
     if (e.code === 401) signOutWithGoogle()
 
-    const message =
-      e.code === 404 ? NOT_FOUND_MESSAGE : UNEXPECTED_ERROR_MESSAGE
-
-    setFlashProps({
-      hidden: false,
-      type: 'error',
-      message,
-    })
+    if (Array.isArray(e.message)) {
```

Comment:
Not sure how I feel about the fact that we are implicitly assuming it is a 422 error based on the fact that the message is an array. This could lead to errors if we introduce other error types that have message arrays.